pkg/rhel/openvdc.spec

```diff
@@ -85,6 +87,13 @@ This is a 'stub'. An appropriate message must be substituted at some point.
 /opt/axsh/openvdc/bin/openvdc-executor
 %dir /etc/openvdc
 %config(noreplace) /etc/openvdc/executor.toml
+%config(noreplace) /etc/firewalld/services/vdc-executor.xml
+
+%post executor
+%{firewalld_reload}
```

Comment:
This line scares me. Doesn't this mean that the port will be opened automatically after installation? I don't think we can just do that to people's servers. I think it can be helpful to put the services in place but I still think it should be up to the admin to explicitly activate them.